plugins/UsersManager/AuthTokenNotifications/AuthTokenProviderInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\UsersManager\AuthTokenNotifications;
+
+interface AuthTokenProviderInterface
```

Comment:
This class should better be called `TokenProviderInterface`, as this should be implemented by other plugins, which might not provide AuthTokens but other tokens.